ci_scripts/template.rst

```diff
@@ -0,0 +1,34 @@
+{# This is a custom template for towncrier md #}
```

Comment:
Could the filename give a hint that the template is for towncrier and markdown and not restructured text 'rst'?